ddtrace/contrib/aiomysql/patch.py

```diff
@@ -21,7 +21,7 @@
 
 CONN_ATTR_BY_TAG = {
     net.TARGET_HOST: "host",
-    net.TARGET_PORT: "port",
```

Comment:
Looks good to me! To fix the commit message, we use conventional commits. See [here](https://www.conventionalcommits.org/en/v1.0.0/). For merging, we may want to reach out to Python core team and see if they're okay with switching just the port tag for now (leaving `out.host`). If they are, we can just switch the target branch to 1.x instead. The rest of networking tags are blocked by security changes to come.

Reply:
> we may want to reach out to Python core team and see if they're okay with switching just the port tag for now (leaving out.host)

👍🏻 to lowering scope to make PR smaller/easier to merge. Especially if it can be merged as-is without any other dependencies with other changes.

Reply:
Rebased and targeted `1.x`